patches/@nuxtjs/markdownit+1.2.1.patch

```diff
@@ -0,0 +1,15 @@
+patch-package
+--- a/node_modules/@nuxtjs/markdownit/index.js
++++ b/node_modules/@nuxtjs/markdownit/index.js
+@@ -17,7 +17,10 @@ module.exports = function nuxtMarkdownit (options) {
+     if (vueLoader.query && vueLoader.query.loaders) {
+       vueLoader.query.loaders['md'] = markDownItLoader
+     } else {
+-       // Sets options loaders (>= rc6)
++      // Sets options loaders (>= rc6)
++      if (!vueLoader.options.loaders) {
++        vueLoader.options.loaders = {}
++      }
```

Comment:
@teodorescuserban just FYI: this patch is the only remarkable aspect of the PR. It's not something I typically do with npm but the fix has been merged for hour months, but the monorepo which gets published to npm hasn't been updated.

I figure holding our own patch makes the most sense. It's a very small modification and the patch must be reevaluated with each release of the parent package. So at some point we can just drop the patch (and the `patch-package` module which applies it to the codebase)

Reply:
Sounds good in principle. Might be daunting on the long run.

Reply:
my sincere hope is that this patch can be removed in a matter of weeks as Nuxt 2.0 becomes the norm rather than the brand new release of their product. the library in question has already been fixed, it's just that it gets published as a mono-repo which naturally lags behind the official repo